vms/proposervm/scheduler/scheduler.go

```diff
@@ -101,9 +99,17 @@ waitloop:
 }
 
 func (s *scheduler) SetBuildBlockTime(t time.Time) {
+	if s.shutdown.Get() {
+		return
+	}
+
 	s.newBuildBlockTime <- t
 }
 
 func (s *scheduler) Close() {
-	close(s.newBuildBlockTime)
```

Comment:
Shouldn't we keep this diff?

Reply:
@joshua-kim  I don't think so. We don't really need to close channels (no leak, GG will take care of them) and the program will panic if it attempts to send on a closed channel.
I believe it's safest to signal closing with quit channel only